optuna_dashboard/_cached_extra_study_property.py

```diff
@@ -54,6 +52,9 @@ def union(self) -> SearchSpaceListT:
         return union
 
     def update(self, trials: List[FrozenTrial]) -> None:
+        if self.has_intermediate_values:
+            return
```

Comment:
Even if `self.has_intermediate_values` is False, we need to continue for-loop to update the search space cache.